ground-core/pom.xml

```diff
@@ -248,6 +265,23 @@
                     </archive>
                 </configuration>
             </plugin>
+
+            <plugin>
+                <groupId>com.spotify</groupId>
```

Comment:
This builds the docker image if you run   
`mvn clean package docker:build`  
We could also add docker:build in one of the lifecycle phases. So that it automatically gets built and pushed. We can also remove the plugin and do it manually if we want fewer dependencies.
[docker maven plugin](https://github.com/spotify/docker-maven-plugin)